avm/res/compute/gallery/main.bicep

```diff
@@ -316,3 +281,38 @@ type imageType = {
   @sys.description('Optional. Describes the disallowed disk types.')
   excludedDiskTypes: string[]?
 }
+
+import { customActionType } from './application/main.bicep'
+type applicationsType = {
+  @sys.description('Required. Name of the application definition.')
+  @minLength(1)
+  @maxLength(80)
+  name: string
+
+  @sys.description('Required. The OS type of the application.')
+  supportedOSType: 'Linux' | 'Windows'
+
+  @sys.description('Optional. The description of this gallery application definition resource. This property is updatable.')
+  description: string?
+
+  @sys.description('Optional. The Eula agreement for the gallery application definition.')
+  eula: string?
+
+  @sys.description('Optional. The privacy statement uri.')
+  privacyStatementUri: string?
+
+  @sys.description('Optional. The release note uri. Has to be a valid URL.')
+  releaseNoteUri: string?
+
+  @sys.description('Optional. The end of life date of the gallery application definition. This property can be used for decommissioning purposes. This property is updatable.')
+  endOfLifeDate: string?
+
+  @sys.description('Optional. Array of role assignments to create.')
+  roleAssignments: roleAssignmentType[]?
```

Comment:
@AlexanderSehr Do you mean here? If yes, do you want to see the import above the UDT definition or above the property inside the UDT? Also, the `roleAssignmentType` will be imported twice.

Reply:
Import definition inside the UDT is not possible. It throws an error.

Reply:
Hey @johnlokerse, not quite. In fact in the template I commented in. Will attach a screenshot :)